source/css/_custom.scss

```diff
@@ -794,5 +794,65 @@ ul.participants.list {
 			display: none;
 		}
 	}
+}
+
+// Eloqua SIG form iframe
+#elq-iframe-SIG-container {
+  iframe {
+	  width: 100%;
+		height: 360px;
+		border: none;
+  }
+}
+
+// tigther spacing between the signup form and upcoming events
+#join > #try > .padding_bottom {
+	padding-bottom: 10px;
+}
+
+// event calendar custom button
+.more-btn {
+  padding-top: 20px;
+
+  &> a {
+	  font-size: 18px;
+  }
+}
+
+// blog section
+#blog {
+	background-color: rgb(245, 245, 245);
```

Comment:
Please use the `info_resalt` class to apply the gray background as used on other sections.

Reply:
Done. The white-gray-white section ordering is "broken" by the first two sections - they need white background, the form and calendar look bad on gray. Then, blog section follows, which looks ok with the gray background and I kinda liked it to be merged with the testimonials in gray, so I did broke it intentionally, to make the participants stand out a bit, but the ordering could definitely be arranged alternatively. Please ping me to discuss the layout.